TEAMS_ROLES_STRUCTURES.md

```diff
@@ -13,21 +13,17 @@
   - [Working Group Captain](#working-group-captain)
   - [Working Group Technical Project Manager](#working-group-technical-project-manager-tpm)
 - [Active Teams](#active-teams)
-  - [**Working** Groups](#working-groups)
-    - [**Project**](#project)
-    - `Development and maintenance of the Protocol:`
-      - [**JavaScript Core Dev**](#javascript-ipfs-implementation)
-      - [**Golang Core Dev**](#golang-ipfs-implementation)
-      - [**IPFS Infrastructure**](#ipfs-infrastructure)
-    - `Efforts to simplify use of the Protocol:`
+  - [**Working Groups**](#working-groups)
+    - [**Project Operations**](#project-operations)
+    - [**Testing Infra**](#testing-infra)
+    - [**IPFS as a Service**](#ipfs-as-a-service)
+    - [**Package Managers**](#package-managers)
+    - [**Docs & Developer UX**](#docs--developer-ux)
```

Comment:
I'm the perpetrator of the Developer UX name - I want to hold us to this because the end aim for this group is *making IPFS easy to use for developers trying to adopt it*. A large part of that is various folds of documentation (aka - accurately communicating what the protocol does), however it could also extend into CLI help text, improved website and forum structure, and websites like protoschool that onboard people onto the core concepts of the dweb through workshops and meetups. I think if we narrowly define the surface area where our user experience happens, we run the risk of limiting our own thinking of how best to solve the problems we identify.

Reply:
👍🏼 that sounds *great* as a vision for this group!

I think a later change after this lands could be the 1-sentence mission statement for each group. I ad-hoc cobbled some version of that together in https://github.com/protocol/pl-ipfs-team/issues/49, but it might help to have one there.

The surprise bit might be the unilateral name change... if the team itself doesn't know it happened. Some communication of this to them would be good if they're not yet aware.

Reply:
The DevUX part actually isn't new - it's been in the name of this team from the beginning (though morphed around a little bit) and we discussed the vision+name for the group in our first task force meeting to make sure folks understood the objective. =]

+1 to mission statements! would be a nice addition

Reply:
added or updated mission statements for each group